pkg/apis/flowcontrol/types.go

```diff
@@ -88,10 +88,11 @@ type FlowSchemaSpec struct {
 	PriorityLevelConfiguration PriorityLevelConfigurationReference
 	// `matchingPrecedence` is used to choose among the FlowSchemas that match a given request. The chosen
 	// FlowSchema is among those with the numerically lowest (which we take to be logically highest)
-	// MatchingPrecedence.  Each MatchingPrecedence value must be non-negative.
+	// MatchingPrecedence.  Each MatchingPrecedence value must be ranged in [0,10000]. For two FlowSchemas
+	// with conflicting MatchingPrecedence, the one created earlier is prior.
```

Comment:
This mechanic can result in inconsistent behavior between clusters where different controllers are maintaining the conflicting values.  You can end up with some clusters "working" and some clusters failing.  I would expect the secondary sort to be consistent based on the manifests, so name is natural.